### PR TITLE
[FEAT]: enforce enterprise secrets license

### PR DESCRIPTION
**Pull request by arturfalcao**
### Description

> Enforcing enterprise secrets license in order to run 2ms

### CODE REVIEW

File: internal/wrappers/jwt-helper.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ type JWTStruct struct {
 	jwt.Claims
 }
 
-var enabledEngines = []string{"sast", "sca", "api-security", "iac-security", "scs", "containers"}
+var enabledEngines = []string{"sast", "sca", "api-security", "iac-security", "scs", "containers", "enterprise-secrets"}
```
Comment (AlvoBen) [inline internal/wrappers/jwt-helper.go:23]: I'm not sure I understand. enterprise-secrets is twoms? if so, what does the "scs" engine type stand for?

File: internal/commands/scan_test.go [hunk 2]
```diff
@@ -689,7 +689,7 @@ func TestAddSCSScan_ResubmitWithOutScorecardFlags_ShouldPass(t *testing.T) {
 		},
 	}
 
-	result, _ := addSCSScan(cmdCommand, resubmitConfig)
+	result, _ := addSCSScan(cmdCommand, resubmitConfig, true)
```
Comment (AlvoBen) [inline internal/commands/scan_test.go:692]: please also add unit tests that check the logic when enterprise license is false

File: internal/wrappers/jwt-helper.go (hunk 1 above)
Comment (arturfalcao) [reply internal/wrappers/jwt-helper.go:23]: Enterprise-secrets is a sublicense of SCS meaning that you can only have Enterprise-secrets license if you already have SCS license

File: internal/commands/scan_test.go (hunk 2 above)
Comment (arturfalcao) [reply internal/commands/scan_test.go:692]: done

File: internal/commands/result.go [hunk 3]
```diff
@@ -1153,6 +1153,8 @@ func countResult(summary *wrappers.ResultSummary, result *wrappers.ScanResult) {
 			} else {
 				return
 			}
+		} else {
+			return
```
[the remark this answers is not in the document]
Comment (arturfalcao) [reply internal/commands/result.go:1157]: this was for the issue we found today, your PR will fix the resutls issue... removing this

**Review by AlvoBen — APPROVED** [review]

**Pull request merged**